### DISCUSSION TITLE
[Q&A] Adding Arabic Language Support

### DISCUSSION THREAD

**Discussion by MEClouds** [lobehub/lobehub#994] [Development Question] [answered]
Hello,

I am interested in contributing to the project and adding Arabic language support. 

Could you please provide guidance or direct me to documentation on adding a new language to the project? I appreciate your assistance.

Thank you!

**Comment by arvinxx** [comment, marked as answer]
Please check the New Locale Guide https://github.com/lobehub/lobe-chat/wiki/Add-New-Locale
